Servers/UI/OJS.Servers.Ui/ClientApp/src/utils/dates.ts

```diff
@@ -21,25 +21,38 @@ const formatDate = (
     ? preciseFormatDate(date, formatString)
     : moment(date).utc(true).local().fromNow());
 
+const getLocalDateTimeInUTC = () => {
+    const now = new Date();
+    return new Date(now.getTime() + now.getTimezoneOffset() * 60000);
+};
```

Comment:
I don't get the intention of this method. "get local date time in UTC" is oxymoron. You either get the local time, or the UTC time. From the implementation I see that you get the local time. Why UTC appears in the name, did you intent something else?

Reply:
Point taken, Renaming to 'GetCurrentTimeInUTC'